cove_bods/templates/cove_bods/additional_checks_table.html

```diff
@@ -569,10 +566,10 @@
             {# Currently this applies to 0.4+ #}
             <tr>
                 <td>
-                    {% blocktrans %}securitiesListings contains objects but hasPublicListing is false. hasPublicListing must be true if securitiesListings are present.{%endblocktrans%}
+                    {% blocktrans %}<code>hasPublicListing</code> has incorrect value. Value of <code>securitiesListings</code> suggests <code>hasPublicListing</code> must be 'true'.{% endblocktrans %}
```

Comment:
"suggests" reads as a little bit uncertain - is this the intention? maybe "indicates" if we want something more forceful

Reply:
I'll leave as is, since with errors like this where x is incompatible with y, we don't know whether x is correct and y wrong, or the other way around.